Glimpse.ApplicationInsights.Sample/App_Start/BundleConfig.cs

```diff
@@ -8,18 +8,8 @@ public class BundleConfig
         // For more information on Bundling, visit http://go.microsoft.com/fwlink/?LinkId=254725
         public static void RegisterBundles(BundleCollection bundles)
         {
-            bundles.Add(new ScriptBundle("~/bundles/jquery").Include(
-                "~/Scripts/jquery-{version}.js"));
-
-            bundles.Add(new ScriptBundle("~/bundles/jqueryval").Include(
-                "~/Scripts/jquery.unobtrusive*",
-                "~/Scripts/jquery.validate*"));
-
-            // Use the development version of Modernizr to develop with and learn from. Then, when you're
-            // ready for production, use the build tool at http://modernizr.com to pick only the tests you need.
-            bundles.Add(new ScriptBundle("~/bundles/modernizr").Include(
-                "~/Scripts/modernizr-*"));
-
+            bundles.Add(new ScriptBundle("~/bundles/jquery"));
```

Comment:
Can you please explain why you are doing this change? Why you don't need versioned jquery and modernizer? I'm curious